sdk/build_c

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+NAME="$(basename "$0")"
+
+print_info() {
+  echo "$NAME is a CLI tool to compile your OSle programs."
+  echo "Feed it an *.c file to compile it into a binary you can run with OSle disk."
+  echo "Additional resources:"
+  echo "  - https://github.com/shikaan/osle to get started with OSle"
+  echo "  - osle-pack to add your program to an OSle image"
+  echo ""
+  print_usage
+}
+
+print_usage() {
+  printf '%s\n    %s\n\n' "Usage:" "$NAME <your_source.c path>"
+  echo "Use \"$NAME --help\" for more information."
+  exit 1
+}
+
+if echo "$1" | grep -Eq '^(-h|--help)$'; then
+  print_info
+fi
+
+filename=$1
+# Check if the file exists
+if [ -z "$filename" ] || [ ! -f "$filename" ]; then
+  echo "$NAME: unable to open ${filename}"
+  print_usage
+fi
+
+base="$(basename "$filename" .c)"
+obj="${base}.o"
+
+# Build the raw binary
+gcc -ffreestanding -nostdlib -nodefaultlibs -Isdk -c "${filename}" -o "${obj}"
```

Comment:
I managed to compile this on MacOS with Apple Silicon with `i686-elf-gcc`, I would assume we need something similar also for Debian on ARM.

We need to update the README.md to mention `gcc` and `i686-elf-gcc` and this script should be adapted to use the correct one depending on the case.

Happy to help with that, if you want me to